### ISSUE TITLE
Sitemap

### ISSUE THREAD

**Issue by ejhumphrey** [cosmir/open-mic#24]
At some point we'll want to hash out what our sitemap actually looks like, what goes on the pages, what's the navigation flow, etc. I imagine we could follow http://image-net.org/ pretty closely and do okay.

So, if https://cosmir.github.io/open-mic/ is the home landing page, I could imagine also ...
- /about
- /annotate
- /login
- /downloads
- /challenges/
-- /2017/leaderboard
-- /2017/submission
- /publications.html (eventually?)

This is super hand-wavy, as I haven't given it much thought. Please offer suggestions if you have them!

**Comment by julian-urbano** [comment]
lgtm

I'm up for starting this and have something to show next week at mediaeval. My vote goes for the option of having the site under `docs/` (github settings). We can use github's Jekyll system to set this up and use markdown. Sounds good?

**Comment by ejhumphrey** [comment]
I have literally **zero** experience with this, and it sounds like you do ... congratulations on your new assignment! 😉 

but srsly, do whatever make sense to you. only comment I'd offer is that maybe we put the website under either `web` or `site`, and reserve `docs` for source documentation?

**Comment by ejhumphrey** [comment]
dang, I see now that our options are only repository root or `docs` ... suppose the latter is the better choice. I'll update that in the settings.

**Comment by julian-urbano** [comment]
There's also the option of using a branch named `gs_pages` or something like that, but imho the easiest thing to do is `docs/`, so that the rest of the files are not accessible from the web browser (not that it really matters anyway). 

We should also have https://cosmir.github.io as main site for the whole thing (needs repo `cosmir.github.io` for the files) with all the main info, and from there point to challenges, such as https://cosmir.github.io/open-mic/ (files in this repo). That way it's easier to separate global stuff from challenge-specific stuff.

**Comment by ejhumphrey** [comment]
agreed, https://cosmir.github.io is a great idea! feel free to tackle that while you're at it prep'ing for MediaEval.

**Comment by ejhumphrey** [comment]
oh, btw, I've pushed a `docs` folder w/index to master and updated the settings, so it's ready for folks to make some sawdust.

**Comment by julian-urbano** [comment]
great. Please create the repo `cosmir.github.io` and I'll start on it. Since that one will only be for the website, I say we do it from the root of the master branch.

**Comment by ejhumphrey** [comment]
done! I think

**Comment by julian-urbano** [comment]
Alright, up and running. Will create actual content tomorrow.
